nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-client-service-api/src/main/java/org/apache/nifi/distributed/cache/client/DistributedMapCacheClient.java

```diff
@@ -105,6 +105,23 @@
      */
     <K, V> void put(K key, V value, Serializer<K> keySerializer, Serializer<V> valueSerializer) throws IOException;
 
+    /**
```

Comment:
" for a set of keys and values of the same types, with the same serialization."  maybe?